src/http_proxy_io.c

```diff
@@ -922,6 +936,56 @@ static int http_proxy_io_set_option(CONCRETE_IO_HANDLE http_proxy_io, const char
     return result;
 }
 
+/*this function will clone an option given by name and value*/
+static void* http_proxy_io_clone_option(const char* name, const void* value)
```

Comment:
const char* const name,

Reply:
I don't see how this is actually "cloning" anything?

Reply:
That is different than the function pointer definition for OptionHandler. Will keep as is unless we change those fps.